src/recipe/Baked.Recipe.Service.Application/Theme/Admin/Components.cs

```diff
@@ -45,32 +45,46 @@ public static ComponentDescriptorAttribute<DataTable> DataTable(
         int? rowsWhenLoading = default,
         string? scrollHeight = default,
         DataTable.Footer? footerTemplate = default,
+        DataTable.Export? exportOptions = default,
         IData? data = default
-    ) => new(
-        new()
+    )
+    {
+        if (exportOptions is not null)
         {
-            Columns = [.. columns ?? []],
-            DataKey = dataKey,
-            ItemsProp = itemsProp,
-            Paginator = paginator,
-            Rows = rows,
-            RowsWhenLoading = rowsWhenLoading,
-            ScrollHeight = scrollHeight,
-            FooterTemplate = footerTemplate
+            exportOptions.ButtonIcon ??= "pi pi-external-link";
         }
-    )
-    { Data = data };
+
+        return new(
+            new()
+            {
+                Columns = [.. columns ?? []],
+                DataKey = dataKey,
+                ItemsProp = itemsProp,
+                Paginator = paginator,
+                Rows = rows,
+                RowsWhenLoading = rowsWhenLoading,
+                ScrollHeight = scrollHeight,
+                FooterTemplate = footerTemplate,
+                ExportOptions = exportOptions
+            }
+        )
+        { Data = data };
+    }
 
     public static DataTable.Column DataTableColumn(string prop, IComponentDescriptor component,
         string? title = default,
-        bool minWidth = false
-    ) => DataTableColumn(prop, component: Conditional(fallback: component), title: title, minWidth: minWidth);
+        bool minWidth = false,
+        bool exportable = false,
+        string? exportHeader = default
```

Comment:
let's not allow overriding export header. if it's exportable, it should use column title as export header